recovery/src/main/java/com/kin/ecosystem/recovery/restore/view/RestoreEnterPasswordFragment.java

```diff
@@ -163,4 +165,11 @@ public void invalidQrError() {
 		contentText.setTextColor(ContextCompat.getColor(getContext(), R.color.kinrecovery_red));
 		password.setFrameBackgroundColor(R.color.kinrecovery_red);
 	}
+
+	@Override
+	public void onDestroy() {
+		super.onDestroy();
+
+		textWatcherAdapter.release();
```

Comment:
should be called before super.

Reply:
why? don't think there is some particular guide regarding that.
From looking at base class (support v4 fragment) super.onDestory merely releasing loader if any (which we don't have, so it does nothing), also looking at `AppCompatActivity` code, there's nothing I can say that must happen at the end. (removing internal handlers callbacks, flagging that activity is destoryed etc.0

Reply:
Well, i've noticed that could happened that you resources won't be GC'ed.
But if you sure 👍